### ISSUE TITLE
[WIP] Mobile Accessibility - CPW

### ISSUE THREAD

**Issue by nickttng** [A11YChi/meetups-and-events#16]
**Why:** As an attendee, I want to understand WCAG 2.1 success criteria for mobile accessibility and an overview of android and iOS accessibility features and tools

- **Title:** It's Always Sunny in Mobile Accessibility
- **Synopsis:** Do you want to be a Golden God of mobile accessibility? First you are going to need your tools. The increased usage of mobile devices by individuals with disabilities makes it critical for teams to understand how to test mobile applications. Currently, the Web Content Accessibility Guidelines (WCAG) addresses only a fraction of mobile web and native application issues and largely anchors them to the standards for non- mobile sites and applications. Putting teams in the difficult position of having to decipher and validate their mobile accessibility approach on their own. In this talk we will discuss how WCAG can be applied to mobile web and native applications, best practices and an overview into accessibility features and tools available for both Android and iOS devices.

---

## ToDo

**30 days before the event**
- [x] Co-org to reach out to presenter
- [x] Have a kick-off conversation
- [x] Coordinate with co-orgs for date availability
- [x] Coordinate with the presenter for next steps
  - [x] Get synopsis and bio
  - [x] Set potential dates
  - [x] Share 'Accessible Presentation' guideline
- [x] Secure ACS captioner
- [x] Create event page/URL

**15 days before the event**
- [x] Announce through channels (MTLS)
  - [x] Mailing list
  - [x] Twitter
  - [x] LinkedIn
  - [x] Web-A11Y Slack
- [x] Share event page/URL with presenter
- [x] If feasible, request (rough) slides from presenter for _initial_ accessibility check

**1-5 days before the event**
- [x] Reminders through channels (MTLS)
  - [x] Mailing list
  - [x] Twitter
  - [x] LinkedIn
  - [x] Web-A11Y Slack
- [x] Follow up with presenter for being online (5-10 before official "online") and any needs
- [x] Update housekeeping slides
- [x]  Presenter's slides
  - [x] Request slides from presenter for _final_ accessibility check
  - [x] Send presenter's slides to ACS

**Day of event**
- [x] Co-orgs are in video meeting 5-10 minutes to discuss any needs/issues before the official "online" for attendees
- [x] Coordinate with presenter
- [x] Announcements and talk

**1-5 days after the event**
- [x] Post-announcement, YouTube share, and thanks (MTLS)
  - [x] Shoutout for presenter
- [x] Finalize transcript
  - [x] Get transcript from ACS
  - [x] Edit transcript and push to GitHub transcript repo

**Post-event (based on team bandwidth)**
- [x] Add transcript to YouTube video talk

**Comment by nickttng** [comment]
This event has been wrapped up and transcript is uploaded to the transcripts repository and shared in YouTube talk's description. We are closing this issue. :tada:
